cpu/avr/radio/rf230bb/rf230bb.c

```diff
@@ -1519,6 +1545,14 @@ rf230_set_pan_addr(unsigned pan,
     PRINTF("\n");
   }
 }
+
+/* From ISR context */
+void
+get_last_rx_packet_timestamp(void)
```

Comment:
This looks like a global symbol and probably needs renamed to `rf230bb_get_last_rx_packet_timestamp()` to match the naming convention.